src/components/__tests__/SfacExtensionSellNotification.test.js

```diff
@@ -36,37 +46,75 @@ describe('SfacExtensionSellNotification component', () => {
     expect(acceptButton.text()).toEqual('Maybe later')
     acceptButton.simulate('click')
 
-    expect(mockProps.onNo).toHaveBeenCalled()
+    expect(CreateSfacExtensionPromptResponseMutation).toHaveBeenCalledWith(
+      mockProps.userId,
+      'chrome',
+      false
+    )
     expect(wrapper.find(Notification).first().prop('open')).toEqual(false)
   })
 
-  it('calls mutation, handler and closes on clicking yes', () => {
+  it('calls mutation and redirect on clicking yes', async () => {
+    expect.assertions(6)
     const SfacExtensionSellNotification =
       require('src/components/SfacExtensionSellNotification').default
     const mockProps = getMockProps()
     const wrapper = mount(<SfacExtensionSellNotification {...mockProps} />)
     expect(wrapper.find(Notification).first().prop('open')).toEqual(true)
     const acceptButton = wrapper.find(Button).at(1)
     expect(acceptButton.text()).toEqual("Let's do it!")
-    acceptButton.simulate('click')
 
-    expect(mockProps.onYes).toHaveBeenCalled()
+    CreateSfacExtensionPromptResponseMutation.mockResolvedValue({})
+
+    await act(async () => {
+      acceptButton.simulate('click')
+
+      await flushAllPromises()
+      jest.runAllTimers()
+      await flushAllPromises()
+    })
+
+    wrapper.update()
+    expect(CreateSfacExtensionPromptResponseMutation).toHaveBeenCalledWith(
+      mockProps.userId,
+      'chrome',
+      true
+    )
     expect(wrapper.find(Notification).first().prop('open')).toEqual(false)
+    expect(windowOpenTop).toHaveBeenCalledWith(GET_SEARCH_URL)
+    expect(logger.error).not.toHaveBeenCalled()
   })
 
-  it('default handlers do not throw', () => {
+  it('calls a redirect to the search engine result page (and does not throw or log) if LogSearchMutation takes a really long time to resolve', async () => {
+    expect.assertions(6)
     const SfacExtensionSellNotification =
       require('src/components/SfacExtensionSellNotification').default
     const mockProps = getMockProps()
-    delete mockProps.onNo
-    delete mockProps.onYes
+    const wrapper = mount(<SfacExtensionSellNotification {...mockProps} />)
+    expect(wrapper.find(Notification).first().prop('open')).toEqual(true)
+    const acceptButton = wrapper.find(Button).at(1)
+    expect(acceptButton.text()).toEqual("Let's do it!")
 
-    let wrapper = mount(<SfacExtensionSellNotification {...mockProps} />)
-    const onNoButton = wrapper.find(Button).at(0)
-    expect(() => onNoButton.simulate('click')).not.toThrow()
+    CreateSfacExtensionPromptResponseMutation.mockRejectedValue(
+      new Error('Uh oh.')
+    )
```

Comment:
Is this testing an error rather than a very long mutation resolution? Clarify either code or test description.